cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -236,4 +240,22 @@ public void renameOutlet(String officeId, String oldOutletId, String newOutletId
             CWMS_OUTLET_PACKAGE.call_RENAME_OUTLET(DSL.using(conn).configuration(), oldOutletId, newOutletId, officeId);
         });
     }
+
+    private static class LocGroupReturn {
```

Comment:
Unneeded maintenance burden adding an inner class like that, your method can just return the LocationGroup itself and let the mapToOutlet function pull out the necessary parts

Reply:
Reworked to return LocationGroup object, mapping occurs in mapToOutlet